manuscript/chapter3.md

```diff
@@ -57,13 +57,25 @@ X> ### Exercise 1
 X>
 X> Convince yourself that the expression {$$}\lambda f \ x . f \ x{/$$} is a well-formed expression by writing down each one of the grammar rules used.
 
-### 3.1.1. Terms reduction
+### 3.1.1. Term reduction
+
+Every variable in a lambda expression can be characterized as either _free_ or _bound_ in that expression.
 
 I> ### Definition 5
 I>
-I> There are two types of variables in the lambda calculus: free and bound. Bound variables are those who appear within the {$$}\lambda{/$$} abstraction. Analogously, free variables are those who do not appear in the {$$}\lambda{/$$} abstraction.
+I> A variable in a lambda expression is called _free_ unless it appears inside at least one lambda expression where it is equal to the argument, and in this case we say the variable is _bound_ at the innermost such lambda.
```

Comment:
This is very correct, but I think it can use some rewording for easier understanding. Maybe:
```
I> A variable in a lambda expression is called _free_ if it does not it appear inside at least one lambda body where it is found in the abstraction. Alternatively, if it does appear inside at least one body, then the variable is _bound_ at the innermost such lambda abstraction.
```